lib/_stream_writable.js

```diff
@@ -144,6 +145,9 @@ function Writable(options) {
 
     if (typeof options.writev === 'function')
       this._writev = options.writev;
+
+    if (typeof options.end === 'function')
+      this._end = options.end;
```

Comment:
Can we use a Symbol here instead of `this._end`?

The thought is that we can encourage new-style constructor use while also avoiding overloading a property that might be in use in the wild.